encoding/serialization.go

```diff
@@ -14,6 +15,10 @@ func (c *Frame) Serialize() ([]byte, error) {
 
 func (c *Frame) Deserialize(data []byte) (*Frame, error) {
 	err := decode(data, c)
+	if !c.Proof.IsInSubGroup() {
+		return nil, fmt.Errorf("proof is in not the subgroup")
```

Comment:
is not `in` the subgroup

Reply:
right, we reject it if it is not in the subgroup

Reply:
I'm pointing out a typo:)